scripts/mirror.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copy all files from the public sources in a catalog the current deployment's
+mirroring bucket. The actual file-copying is not yet implemented, so all this
+currently does is send messages to the indexer app that don't do anything.
```

Comment:
Consider adding a FIXME or reference to #6856 ?

Reply:
https://github.com/DataBiosphere/azul/pull/6905#discussion_r2003819203